### ISSUE TITLE
longtable can't handle a pagebreak before a multicolumn

### ISSUE THREAD

**Issue by teepeemm** [brucemiller/LaTeXML#2000]
The tex code
```tex
\documentclass{article}
\usepackage{longtable}
\begin{document}
\begin{longtable}{c}
\pagebreak
\multicolumn{1}{c}{text}
\end{longtable}
\end{document}
```
under `latexml` has the console output:
```
Error:unexpected:\omit \omit cannot be used here at script.tex; line 6 col 0
Error:unexpected:\endgroup Attempt to close non-boxing group at script.tex; line 7 col 0
Warning:unexpected:\end{document} Attempt to end document with open groups, environments or conditionals at script.tex; line 8 col 0
Error:unexpected:<endgroup> Attempt to pop last locked stack frame at String; line 0 col 0
Error:malformed:ltx:document Closing </ltx:document> whose open descendents do not auto-close at String; line 0 col 0
Error:malformed:ltx:table Attempt to close </ltx:table>, which isn't open
Conversion complete 1 warning; 5 errors
```
and in the xml, the page break has escaped to after the tabular, but within the table.  If a non-multicolumn comes after the page break, then the conversion has no errors or warnings, but the pagebreak escapes to after the tabular and after the table.  Since we don't do much with `\pagebreak` anyway, would it be better to have `\pagebreak` produce no xml output if it's in a longtable?

**Comment by dginev** [comment]
Right, a really quick fix here would be to `Let('\pagebreak',`\relax');` inside the longtable environment.

And maybe in tabulars in general?

**Comment by teepeemm** [comment]
Most tabular environments (I looked at booktabs, deluxetable, longtable, pgfplotstable, supertabular, tablefootnote, tabularx, tabulary, threeparttable) don't span multiple pages to begin with.  deluxetable and supertabular can span multiple pages, but they ignore `\pagebreak` in the general case, and report an error in the `\pagebreak\multicolumn` case.  Since that suggests people won't have them in TeX, I don't know that we need to worry about them.

But if we find ourselves wanting to make sure and ignore the pagebreak, the other thing I realized is that `\pagebreak` takes an optional argument that also needs to be ignored.  Consequently, I added `DefMacro('\@gobble@optional[]', Tokens());` to longtable.sty.ltxml.
